draft-regext-rdap-extensions.md

```diff
@@ -379,6 +381,59 @@ extension registration that attempts to use one of the RDAP
 conformance values given in this section as an extension identifier
 (and so as an RDAP conformance value also) will be rejected.
 
+# Redirects {#redirects}
+
+[@!RFC7480] describes the use of redirects in RDAP. Redirects are prominent
+in the discovery of authoritative INR servers as the process outlined in
+[@?RFC9224], which uses IANA allocations, does not account for transfers of
+resources between INRs. [@!RFC7480, Section 4.3] instructs servers to ignore
+unknown query parameters. As it relates to issuing URLs for redirects, servers
+MUST NOT blindly copy query parameters from a request to a redirect URL as
+query parameters may contain sensitive information, such as security credentials,
+not relevant to the target server of the URL. Following the advice in [@!RFC7480],
+servers SHOULD only place query paramters in redirect URLs when it is known
+by the origin server (the server issuing the redirect) that the target server
+(the server referenced by the redirect) can process the query parameter and the
+contents of the query parameter are appropriate to be received by the target.
+
+As it unlikely that every server in a cross-authority, redirect scenario will be upgraded to process
+every new extension, extensions should not solely rely on query parameters to convey
+information as query parameters are not guaranteed to survive a redirect.
```

Comment:
How about "to convey information about a resource"? Since query parameters help identify a resource, and not convey resource metadata (representation).